front-end-notes/bower.md

```diff
@@ -0,0 +1,37 @@
+
+# Bower 
+Bower is a package manager for the web.
+> Web sites are made of lots of things — frameworks, libraries, assets, utilities, and rainbows. 
+> Bower manages all these things for you.
+> Bower works by fetching and installing packages from all over, taking care of hunting, finding, downloading, and saving the stuff you’re looking for. (Bower.io) 
+
+## Installing Bower
+ Bower is a command line utility. Install it with npm.<br />
+  <br />
+ `$ npm install -g bower`<br />
+ <br />
+*Bower requires node, npm and git.*
+## Installing Packages
+**Install packages individually** <br />
+<br />
+`$ bower intall <package name>`<br />

```

Comment:
Mention the _--save_ option to save the installed package into the _bower.json_ file